geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala

```diff
@@ -146,64 +143,66 @@ class AccumuloDataStore(val connector: Connector, override val config: AccumuloD
     import SimpleFeatureTypes.Configs.{ENABLED_INDEX_OPTS, ENABLED_INDICES}
     import SimpleFeatureTypes.InternalConfigs.{INDEX_VERSIONS, SCHEMA_VERSION_KEY}
 
-    var sft = super.getSchema(typeName)
+    val escapedTypeName = typeName.replace("~", ESCAPE)
```

Comment:
Yeah. After taking a better look at the metadata implementation, that does seem like the better place to handle the tildes.